### DISCUSSION TITLE
(use case): todo-txt-file & github-readme-stats

### DISCUSSION THREAD

**Discussion by ghost** [anuraghazra/github-readme-stats#2140] [Q&A] [answered]
Hi everyone.

I would like to know if it would be possible to read a txt file([todo-txt-file](https://github.com/todotxt/todo.txt)) and display the information in github-readme-stats.

#### What is todo-txt file?
A single line in your todo.txt text file represents a single task.

#### Concept
<img src="https://raw.githubusercontent.com/todotxt/todo.txt/master/description.svg" width="650px"/>

#### Idea/sample-data: todo-txt-file.json
```json
[
  {
		"task_status": "made",
		"task_datetime_start": "2011-03-02",
		"task_datetime_end": "2011-03-01",
		"task_description": "Review Tim's pull request",
		"task_tags": ["TodoTxtTouch"],
		"task_project": ["github"]
	},
	{
		"task_status": "made",
		"task_datetime_start": "2011-03-02",
		"task_datetime_end": "2011-03-01",
		"task_description": "Review Tim's pull request",
		"task_tags": ["TodoTxtTouch"],
		"task_project": ["github"]
	}
]
```

#### URL
```
https://github-readme-stats.vercel.app/api?upload_file=todo-txt-file.txt&read="yes"
```

#### design
![image](https://user-images.githubusercontent.com/111701513/194202526-70192fe8-600d-43b7-a94e-655add28b462.png)

**Comment by rickstaa** [comment, marked as answer]
@codehangen Thanks for your feature request. Since this is quite a specific use case, I don't think we will add support for this on the main GRS instance shortly. If #2151 is merged, you can create your own card repository using GRS common functions and modules.

**Reply by ghost** [reply]
Hi rickstaa, thank you for feedback.
